setup.py

```diff
@@ -37,8 +94,21 @@ def parse_requirements_file(path):
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
-if torch.cuda.is_available():
+# Add C++ and CUDA extensions
+compile_cuda_flag = os.environ.get("THESEUS_ENABLE_CUDA")
```

Comment:
Following up on the discussion in #304, is there a reason why the currently named `THESEUS_ENABLE_CUDA` shouldn't be the default behavior? Does it interact weirdly with `pip install -e .` for non-CUDA machines?

Otherwise, why not make this default, remove the flag completely, and just add another flag that disables all extensions if needed?

Reply:
I would leave `THESEUS_ENABLE_CUDA`, no weird interaction AFAIK, it just overrides the result of `torch.cuda.is_available()`

Reply:
Thanks @maurimo for clarifying offline. Current proposal is to rename as `THESEUS_FORCE_CUDA`.

Reply:
renamed to THESEUS_FORCE_CUDA.